policyengine_us/variables/gov/states/me/tax/income/taxable_income/deductions/me_itemized_deductions_pre_phaseout.py

```diff
@@ -0,0 +1,39 @@
+from policyengine_us.model_api import *
+
+
+class me_itemized_deductions_pre_phaseout(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Maine itemized deductions before phaseout"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/22_1040me_sched_2_ff.pdf",
+        "https://www.mainelegislature.org/legis/statutes/36/title36sec5125.html",
+    )
+    defined_for = StateCode.ME
+
+    def formula(tax_unit, period, parameters):
+        # Whether or not the taxpayer itemized for federal taxes.
+        us_itemizing = tax_unit("tax_unit_itemizes", period)
+
+        # Get the Maine itemizing deduction parameters.
+        p = parameters(period).gov.states.me.tax.income.deductions.itemized
+
+        # Get federal itemized deductions.
+        deduction = tax_unit("taxable_income_deductions_if_itemizing", period)
+
+        # Get medical (and dental) expenses.
+        medical_expenses = tax_unit("medical_expense_deduction", period)
+
+        # Get real estate (and property) taxes.
```

Comment:
I don't see it in the form, but 3A of the legal code also says to subtract state and local income tax:
>A. Reduced by any amount attributable to income taxes or sales and use taxes imposed by this State or any other taxing jurisdiction; 

This is necessary to avoid circularity.